pkg/api/openshiftcluster.go

```diff
@@ -160,6 +161,13 @@ const (
 	ProvisioningStateFailed        ProvisioningState = "Failed"
 )
 
+type MaintenanceTask string
+
+const (
+	MaintenanceTaskInfrastructure MaintenanceTask = "Infrastructure"
```

Comment:
Something like "everything" instead of `Infrastructure`. As far I see from a quick look - it runs all the stesps.

Reply:
Yeah, that's a good idea.